src/uwtools/config/tools.py

```diff
@@ -248,48 +282,24 @@ def _validate_depth(
     target_class = format_to_config(target_format)
     config = config_obj.data if isinstance(config_obj, Config) else config_obj
     if bad_depth(target_class.get_depth_threshold(), depth(config)):
-        raise log_and_error(
+        raise UWConfigError(
             "Cannot %s depth-%s config to type-'%s' config" % (action, depth(config), target_format)
         )
 
 
-def _validate_format_output(input_fmt: str, output_fmt: str) -> None:
+def _validate_format(other_fmt_desc: str, other_fmt: str, input_fmt: str) -> None:
```

Comment:
This is now more generic as it handles both output formats and update formats -- so "other" instead of "output".